pack/silk.json

```diff
@@ -396,6 +396,13 @@
         "traits": "Web-Warrior.",
         "type_code": "ally"
     },
+    {
+        "code": "52023",
```

Comment:
Thanks for adding this. This was one of the unknown cards that I couldn't find released yet when https://github.com/zzorba/marvelsdb-json-data/pull/603 was merged.

This code should be 52022 though to be unique within this file and to match the position it is in.

Reply:
Oh yes! Oops. I've fixed that now.